containers/org.eclipse.linuxtools.docker.ui/src/org/eclipse/linuxtools/internal/docker/ui/jobs/CopyFromDockerJob.java

```diff
@@ -240,7 +237,7 @@ public CopyFromDockerJob(IDockerConnection connection, CopyType copyType, String
 			m_image = desc;
 			m_containerId = null;
 		} else {
-			throw new RuntimeException("Unknow parameter");

```

Comment:
Unknown is misspelled.  This kind of error and others should be an assert and throw an AssertionException.  Code will legitimately try/catch RuntimeExceptions for other reasons.  An assert makes sense and you don't have to translate the message plus it is clear it should never happen.